app/connectors/EmailClient.scala

```diff
@@ -198,6 +198,9 @@ trait EmailClient extends WSHttp {
     sendEmail(to, "fset_faststream_adjustments_changed",
       Map("name" -> name, "etrayAdjustments" -> etrayAdjustments, "videoAdjustments" -> videoAdjustments))
 
+  def sendApplicationExtendedToSdip(to: String, name: String)(implicit hc: HeaderCarrier) =
```

Comment:
I would suggest to use the same method name as the email template, i.e.:

def sendApplicationConvertedToSdip(to: String, name: String)(implicit hc: HeaderCarrier) =